code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -277,4 +277,70 @@
 				qdel(hotspot)
 
 		if(reac_volume >= 6)
-			T.freon_gas_act() //freon in my pocket
\ No newline at end of file
+			T.freon_gas_act() //freon in my pocket
+
+/datum/reagent/arclumin//memechem made in honour of the late arclumin
+	name = "Arc-Luminol"
+	id = "arclumin"
+	description = "You have no idea what the fuck this is but it looks absurdly unstable, it emits a healthy glow so ingestion is probably not a good idea."
```

Comment:
"healthy" is kinda not right?? "radiant" might be a better word to use?
Change the comma to a period.
Change "it emits" to "It is emitting"